src/pages/quickstart-steps/FundWallet.jsx

```diff
@@ -28,15 +30,15 @@ export default function FundWallet({ nextStep }) {
     const callApiForFunding = async () => {
         setLoading(true);
         setError('');
-
         try {
-            let res = await fundAddress(address);
+            let res = await get(siteConfig.customFields.TEST_FUND_API + '/fund/' + address);
             if (res.error) {
                 setError(res.error);
             }
             await updateBalance(ctx, address);
             setLoading(false);
         } catch (exception) {
+            console.log(exception);
```

Comment:
Careful Mr. Cat

Reply:
Man, did you see how many I removed though? This is a small price to pay.